src/main/java/Duke.java

```diff
@@ -1,10 +1,70 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Methods.printGreetings();
+
+        Task[] actions = new Task[100];
+        int actionCounter = 0;
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        int taskNumber;
+        String[] decisions;
+        Task toBeAdded;
+        String[] dates;
+        do {
+            line = in.nextLine();
+            decisions = line.split(" ");
+            dates = line.split("/");
+            switch (decisions[0]) {
+                case "echo":
+                    System.out.println(Methods.findTaskDetails(line));
+                    break;
+
+                case "todo":
+                    toBeAdded = new Todo(Methods.findTaskDetails(line));
+                    actions[actionCounter] = toBeAdded;
+                    actionCounter++;
+                    Methods.printAcknowledgement(toBeAdded,actionCounter);
+                    break;
+
+                case "event":
+                    toBeAdded = new Event(Methods.findTaskDetails(dates[0]), Methods.findTaskDetails(dates[1]), Methods.findTaskDetails(dates[2]));
+                    actions[actionCounter] = toBeAdded;
+                    actionCounter++;
+                    Methods.printAcknowledgement(toBeAdded,actionCounter);
+                    break;
+
+                case "deadline":
+                    toBeAdded = new Deadline(Methods.findTaskDetails(dates[0]), Methods.findTaskDetails(dates[1]));
+                    actions[actionCounter] = toBeAdded;
+                    actionCounter++;
+                    Methods.printAcknowledgement(toBeAdded,actionCounter);
+                    break;
+
+                case "mark":
+                    taskNumber = Integer.parseInt(decisions[1]) - 1;
+                    actions[taskNumber].mark();
+                    Methods.printDoneMarkingTasks(actions[taskNumber]);
+                    break;
+
+                case "unmark":
+                    taskNumber = Integer.parseInt(decisions[1]) - 1;
+                    actions[taskNumber].unmark();
+                    Methods.printDoneMarkingTasks(actions[taskNumber]);
+                    break;
+
+                case "list":
+                    if (actionCounter == 0) {
+                        Methods.printEmptyList();
+                        continue;
+                    }
+                    for (int iterator = 0; iterator < actionCounter; iterator++) {
+                        Methods.printListElement(iterator, actions[iterator]);
+                    }
+                    break;
+            }
```

Comment:
Perhaps you could include a default branch for your switch block so in the case where your command is not caught by any of the case statements, your code will still function.